requirements/base.txt

```diff
@@ -10,5 +10,5 @@ requests==2.7.0
 git+https://github.com/rosskarchner/govdelivery.git#egg=govdelivery
 icalendar==3.9.0
 django-localflavor==1.1
-wagtail==1.1
 MySQL-python==1.2.5
+git+https://github.com/torchbox/wagtail.git#egg=wagtail
```

Comment:
I'm having issues with wagtail assets not building with that version. Possibly do `-e git+https://github.com/torchbox/wagtail.git#egg=wagtail` to fix?